ivi/interface/pyvisa.py

```diff
@@ -112,20 +113,20 @@ def trigger(self):
 
     def clear(self):
         "Send clear command"
-        raise NotImplementedError()
+        self.instrument.clear()
 
     def remote(self):
         "Send remote command"
-        raise NotImplementedError()
+        self.instrument.gpib_control_ren(pyvisa.constants.VI_GPIB_REN_ASSERT_ADDRESS)
```

Comment:
In the general case of any VISA device, this doesn't seem correct. What if the instrument is an USB or Ethernet, but talks VISA?

(similar concern for line 124)